db-service/lib/cqn2sql.js

```diff
@@ -336,7 +342,9 @@ class CQN2SQLRenderer {
     const { ref, as } = from
     const _aliased = as ? s => s + ` as ${this.quote(as)}` : s => s
     if (ref) {
-      const z = ref[0]
+      const { target } = SELECT.from(from)
```

Comment:
Why do we have to create a new query to get the target? 
Can't we just take it from the rendered query?